static/gsApp/components/gsBanner.tsx

```diff
@@ -1063,6 +1063,10 @@ class GSBanner extends Component<Props, State> {
         product: DataCategory.SPANS,
         categories: [DataCategory.SPANS],
       };
+      this.PATHS_FOR_PRODUCT_TRIALS['/profiling/'] = {
+        product: DataCategory.PROFILES,
+        categories: [DataCategory.PROFILE_DURATION, DataCategory.PROFILE_DURATION_UI],
+      };
```

Comment:
LGTM
How about adding this directly to `PATHS_FOR_PRODUCT_TRIALS` above?

Reply:
I thought about it but don't AM2 plans have the addition of transaction profiles which are billed as performance units? Not sure how that would work together. I'm open to suggestions